Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN echo "**** install dev packages ****" && \
     mamba install --quiet \
         git \
         python="3.10" \
+        "setuptools<66.0.0a" \
```

Comment:
We should use the repodata patch to conda-build instead of this pin.

Reply:
See https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/387

Reply:
```suggestion
```